mantis-control-plane/mantis-control-plane-client/src/main/java/io/mantisrx/server/master/client/config/StaticPropertiesConfigurationFactory.java

```diff
@@ -27,6 +29,8 @@ public class StaticPropertiesConfigurationFactory implements ConfigurationFactor
     public StaticPropertiesConfigurationFactory(Properties props) {
         delegate = new ConfigurationObjectFactory(props);
         delegate.addCoercible(new MetricsCoercer(props));
+        delegate.addCoercible(new LeaderElectorFactoryCoercer());
+        delegate.addCoercible(new LeaderMonitorFactoryCoercer());
 //        delegate.addCoercible(new MantisPropertiesCoercer(props));
```

Comment:
should this be deleted? As I review the PR it's pops out at me, but during my work I didn't notice this commented code.